yarn.lock

```diff
@@ -9,7 +9,27 @@
   dependencies:
     "@babel/highlight" "^7.0.0"
 
-"@babel/core@^7.1.0", "@babel/core@^7.2.2", "@babel/core@^7.3.4":
+"@babel/core@^7.1.0", "@babel/core@^7.3.4":
```

Comment:
Can you check again why we have changes in the yarn.lock and if we have new dependencies that require OSS scan?

Reply:
I reverted these changes because I don't why they happened 🤔 There are no any changes in any `package.json`